tests/sentry/integrations/msteams/notifications/test_assigned.py

```diff
@@ -28,7 +29,7 @@ def test_assigned(self, mock_send_card: MagicMock):
         with self.tasks():
             notification.send()
 
-        mock_send_card.assert_called_once()
```

Comment:
For reference, I changed this because the linter was yelling about it: 

`Avoid using the assert_called_once mock call as it is confusing and prone to causing invalid test behavior.`